### PR TITLE
Updating to snapshot version and making build tools a static version

### PR DESCRIPTION
**Pull request by shorea**

### CODE REVIEW

File: build-tools/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <groupId>software.amazon.awssdk</groupId>
     <artifactId>build-tools</artifactId>
     <name>AWS Java SDK :: Build Tools</name>
-    <version>2.0.0-preview-4</version>
+    <version>1.0</version>
```
Comment (kiiadi) [inline build-tools/pom.xml]: I tend to agree that we can make this version independently but we should follow the same scheme for all our modules right? It seems strange to me to have build-tools @ `1.0` when everything else is `2.0.0`

**Comment by millems** [comment]
What's our goal for having build-tools versioned independently?

**Comment by shorea** [comment]
> What's our goal for having build-tools versioned independently?

Since it doesn't declare the pom module as a parent it doesn't get it's version updated with the mvn versions command.

**Comment by millems** [comment]
Is there a reason we couldn't make it the parent? Sorry for the simple question - I'm not particularly familiar with the impact of it.

File: build-tools/pom.xml (hunk 1 above)
Comment (shorea) [reply build-tools/pom.xml]: I choose 1.0 so it's different enough to not be confused why it isn't updated.

**Comment by shorea** [comment]
It causes a circular dependency because the parent references the build-tools plugin in it's POM.

**Comment by millems** [comment]
Makes sense :+1:.

**Pull request merged**